templates/init-default/README.md

```diff
@@ -0,0 +1,12 @@
+## Run bot
+
+Run [botpress start](https://botpress.io/docs/reference/cli.html#start) to start your bot.
```

Comment:
Better to reference `npm start`.
Can you also please fix the template's package.json to reference node `< 9` instead of `<= 8.9`?